risc0/build/src/lib.rs

```diff
@@ -490,6 +512,30 @@ fn build_staticlib(guest_pkg: &str, features: &[&str]) -> String {
     }
 }
 
+// HACK: Attempt to bypass the parent cargo output capture and
+// send directly to the tty, if available.  This way we get
+// progress messages from the inner cargo so the user doesn't
+// think it's just hanging.
+fn tty_println(msg: &str) {
```

Comment:
Love this, although related to https://github.com/risc0/risc0/issues/1685

Aside: I think we need better loading /progress information, as spinners don't give an idea if you are stuck or not in many cases, like `rzup` and `cargo risczero install`

Reply:
We don't have that option since we're running inside the context of a cargo build process.